src/shared/enums/ASCNEnums.ts

```diff
@@ -0,0 +1,10 @@
+export enum ASCNAttributes {
```

Comment:
what's the consideration have this enum as a separate file? I see other enum you added in this pr are in constants.ts

Reply:
Most of the changes in constants.ts were made by @sheridancbio, looping him in. But from my understanding the constants.ts were things used as args for API calls. 

ASCNEnums is specifically for the allele_specific_copy_number model in the backend.